tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs

```diff
@@ -106,13 +112,32 @@ data ExternalType
       Show
     )
 
+{- | Different types of value:
+
+  `TokensValue` is only used within `ContextBuilder ('ForMinting r)`
+  for representing classes, belonging to the tested MintingPolicy.
+
+  In all other cases `GeneralValue` is used.
```

Comment:
Haddock gets confused by backticks. If you want to link the documentation to the definitions, you want to use 'TokensValue'; if you want monospacing but not links, use @TokensValue@.